La plus petite charge électrique  1.6E-19

In [4]:
c=299792458;c/1.00058

2.99618679166084e8

In [2]:
c-c/1.00058 # air 173.778 km/s

173778.833916306

In [8]:
9.109E-31*c^2/1.60E-34

5.11672557694604e20

In [10]:
re=2.8179E-15*2*3.1415;re

1.77048657000000e-14

Le rayon de l'électron est 1.77 comme la racine carrée de pi